pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -63,6 +63,10 @@ const (
 	ParamRemoteAddress     = "remote-address"
 	ParamConnectionMethod  = "connection-method"
 	ParamConnectionTimeout = "connection-timeout"
+	ParamDetachable        = "detachable"
```

Comment:
Yeah, I think `detach` is better the way it currently works. Previously it would attach directly, just make it detachable, so for that UX I think _that_ made more sense. I personally like it better with `--detach` and just returning the ID(s) (which is missing currently).